Gemfile

```diff
@@ -1,7 +1,7 @@
 # frozen_string_literal: true
 
 source 'https://rubygems.org'
-ruby '>= 2.3.0', '< 2.5.0'
+ruby '>= 2.4.0', '< 2.6.0'
```

Comment:
In my opinion: No reason to drop support for 2.3.0. Some people may still use it and there's no *critical* reason to upgrade the Ruby runtime

Reply:
okay